### PR TITLE
add missing methods and update readme

### PR DESCRIPTION
**Pull request by stewones**

### CODE REVIEW

File: ios/Plugin/Plugin.swift [hunk 1]
```diff
@@ -76,4 +76,18 @@ public class FirebaseAnalytics: CAPPlugin {
...
+            call.success()
+        }
+    }
+  
+    @objc func disable(_ call: CAPPluginCall) {
+            DispatchQueue.main.async {
+            Analytics.setAnalyticsCollectionEnabled(false)
+            call.success()
+        }
+    }
```
Comment (imhoffd) [inline ios/Plugin/Plugin.swift:92]: Plugin calls already use a dispatch queue within Capacitor, AFAIK. Is this necessary?

**Review by imhoffd — COMMENTED** [review]
Looking good. One thing:

I think we decided on `@capacitor-community/firebase-analytics` for the package name, not `@capacitor-community/analytics`.

File: ios/Plugin/Plugin.swift (hunk 1 above)
Comment (priyankpat) [reply ios/Plugin/Plugin.swift:92]: Can [confirm](https://github.com/ionic-team/capacitor/blob/cdd03370a5f7c9be97c3a7b45e3ca24e9ea95b44/ios/Capacitor/Capacitor/CAPBridge.swift#L425)  that dispatch queue is invoked with js call. What would be the reason for [built in plugins](https://github.com/ionic-team/capacitor/search?p=1&q=DispatchQueue.main.async&unscoped_q=DispatchQueue.main.async) to call dispatch queue separately?

**Comment by priyankpat** [comment]
@stewwan I propose we direct the users to official firebase docs for initial setup across all firebase plugins. What do you think?

[Android](https://firebase.google.com/docs/android/setup)
[iOS](https://firebase.google.com/docs/analytics/get-started?platform=ios)
[Web](https://firebase.google.com/docs/analytics/get-started?platform=web)

**Comment by stewones** [comment]
We can link docs for sure but I think it’s not enough. They aren’t helpful for a quick setup, step by step has proven to work better. I do this for all my plugins and people seem to appreciate. Including myself I prefer to look for a quick guide on README rather searching on google.

File: ios/Plugin/Plugin.swift (hunk 1 above)
Comment (stewones) [reply ios/Plugin/Plugin.swift:92]: Took from my plugin, back on earlier days of capacitor it was necessary.

Comment (stewones) [reply ios/Plugin/Plugin.swift:92]: Not sure if it still necessary, though. From what I remember it crashes the app if we not dispatch async.

**Review by priyankpat — APPROVED** [review]

File: ios/Plugin/Plugin.swift (hunk 1 above)
Comment (priyankpat) [reply ios/Plugin/Plugin.swift:92]: I've just tested it again. Safe to remove the dispatch queue.

**Pull request merged**